### DISCUSSION TITLE
Dashboard is blank

### DISCUSSION THREAD

**Discussion by siljiths** [Sitecore/Sitecore.Commerce.Headstart.ReactAdmin#238] [Q&A] [answered]
I followed the readme file and deployed both locally and to vercel but the dashboard is blank as shown in the attachment in both Vercel and local dev. 
![Admin](https://github.com/Sitecore/Sitecore.Commerce.Headstart.ReactAdmin/assets/149743729/003a0276-2839-4710-8a1b-88f724bac234)

any pointers? 
I assume  the admin client ID is just an API Client ID. I've created a Client ID as below 
![image](https://github.com/Sitecore/Sitecore.Commerce.Headstart.ReactAdmin/assets/149743729/d3a8dcd4-ff23-4f87-9f66-c72fd40b33a4)

**Comment by crhistianramirez** [comment, marked as answer]
Hi @siljiths we made some security profile changes recently and haven't updated the readme, so apologies. To fix this issue do the following:

1. In the API console create a security profile with the role "Full Access"
2. Assign the security profile to the admin user that you are logging in as
3. Log out
4. Log in as your admin user, you should now see everything.
5. When you create new admin users through the UI make sure you assign a security profile. Full access should only be used for demo/development purposes. In production you'll want to create a security profile that only grants the user permissions that they need.

**Comment by siljiths** [comment]
Hi @crhistianramirez ,
That worked fine. Thanks a lot.
Regards,
Siljith
